apps/api/src/app/subscribers/usecases/delete-subscriber-credentials/delete-subscriber-credentials.usecase.ts

```diff
@@ -68,7 +53,7 @@ export class DeleteSubscriberCredentials {
         _id: _subscriberId,
         _environmentId: environmentId,
       },
-      { $pull: { channels: { _integrationId: integrationId } } }
+      { $pull: { channels: { providerId: providerId } } }
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** Use shorthand object reference

```suggestion
      { $pull: { channels: { providerId } } }
```